CHANGELOG.md

```diff
@@ -1,5 +1,33 @@
 # Changelog
 
+## 2.4
+
+* Listeners are now opt-in when not handling API Platform operations
+* `DISTINCT` is not used when there are no joins
+* Preserve manual join in FilterEagerLoadingExtension #2622
+* `elasticsearch` can be disabled resource-wise or per-operation
+* The `messenger` attribute can now take the `input` string as a value (`messeger="input"`). This will use a default transformer so that the given `input` is directly sent to the messenger handler.
```

Comment:
```suggestion
* The `messenger` attribute can now take the `input` string as a value (`messenger="input"`). This will use a default transformer so that the given `input` is directly sent to the messenger handler.
```